### DISCUSSION TITLE
Add a New product logo to paste

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#2640] [Request: Icon] [answered]
Hello there,

Can you please add the IoT> Programmable X - Asset Tracker Product logo to paste so that we can be used for the new Product in the IoT Order flow?

Attached is the logo svg file. 
[programmable-asset-tracker.zip](https://github.com/twilio-labs/paste/files/9427551/programmable-asset-tracker.zip)

Found the instruction to self upload the icon - https://paste.twilio.design/introduction/contributing/icons/#adding-the-icon-to-the-paste-repository

can we use this to upload the product logo as well? How about adding this logos in Figma?

Thank you.

**Comment by SiTaggart** [comment, marked as answer]
Hi @ishah-twilio 

That page outlines both the Figma and Code contribution guides, and applies to both Logos and Product Logos.

Just to double-check, I'm assuming the product logo was provided to us by marketing?

If so, then yeah, feel free to follow the contribution guidelines to get it accepted quicker into the system. Otherwise we'll need to find some time to schedule it into our workload and up coming sprints.

Thanks

**Reply by crubio** [reply]
Hi Simon, followed the [contributing guidelines](https://paste.twilio.design/introduction/contributing/icons/#adding-the-icon-to-the-paste-repository) for adding icons and created a PR for this request: https://github.com/twilio-labs/paste/pull/2657

Let us know if this is correct or if the PR needs anything else.

**Reply by serifluous** [reply]
Thanks, @crubio !

@ishah-twilio Can you make sure you follow the guidelines for setting the icon up in Figma and exporting the SVG written here? https://paste.twilio.design/introduction/contributing/icons/#setting-up-an-icon-in-figma There are a few things that need to be cleaned up in the exported SVG code (e.g., viewBox should be "0 0 20 20", there should be no style or XML tags, etc).

If you start by [creating a branch in Figma](https://help.figma.com/hc/en-us/articles/360063144053-Guide-to-branching) off of the ["Paste Components" file](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components), we can help you out there.

**Reply by crubio** [reply]
Just wanted to check in and see if the icon was added to Paste. I'll follow along on this discussion, unless its moved somewhere else. Thanks!

**Reply by shleewhite** [reply]
Hi! I'm actually working on adding it now: https://github.com/twilio-labs/paste/pull/2690

It should merge soon and I will follow up again when it is released on npm.
